api/src/main/java/edu/wpi/first/shuffleboard/api/util/NetworkTableUtils.java

```diff
@@ -119,6 +119,14 @@ public static boolean isDelete(int flags) {
   }
 
   /**
+   * Checks if the given key is metadata, eg matches the format "~METADATA~" or ".metadata"
+   */
+  public static boolean isMetadata(String key) {
```

Comment:
@PeterJohnson What do you think about moving this method into either ntcore or some other common tool jar?